pipeline/relation_embedder/path_concatenator/src/main/scala/weco/pipeline/path_concatenator/PathOps.scala

```diff
@@ -0,0 +1,15 @@
+package weco.pipeline.path_concatenator
+
+object PathOps {
+  implicit class StringOps(path: String) {
```

Comment:
We should combine this with [PathOps](https://github.com/wellcomecollection/catalogue-pipeline/blob/92193049aefe60f4e44b21bf35fcede2ee4291ca/pipeline/relation_embedder/relation_embedder/src/main/scala/weco/pipeline/relation_embedder/models/PathOps.scala) in the relation embedder, put them both in internal model and share them among applications.

Reply:
I agree, and that its why I chose to keep the same name, so that it would be just a matter of changing the import path once they were merged.

I was reluctant to make internal-only changes in internal model while the version check still exists to keep pipeline and API in harmony, and was wondering if there is a better place to put common pipeline-only code like this.